server/ladder_service.py

```diff
@@ -76,17 +85,27 @@ async def update_data(self) -> None:
             self.ladder_1v1_map_pool.set_maps(maps)
 
             map_pool_maps = await self.fetch_map_pools(conn)
-            matchmaker_queues = await self.fetch_matchmaker_queues(conn)
+            db_queues = await self.fetch_matchmaker_queues(conn)
 
-        for name, map_pools in matchmaker_queues.items():
+        for name, info in db_queues.items():
             if name not in self.queues:
-                self.queues[name] = MatchmakerQueue(
+                queue = MatchmakerQueue(
                     name=name,
+                    featured_mod=info["mod"],
+                    rating_type=info["rating_type"],
+                    team_size=info["team_size"],
                     game_service=self.game_service
                 )
-            queue = self.queues[name]
+                self.queues[name] = queue
+                queue.initialize()
+                asyncio.ensure_future(self.handle_queue_matches(queue))
+            else:
+                queue = self.queues[name]
+                queue.featured_mod = info["mod"]
+                queue.rating_type = info["rating_type"]
+                queue.team_size = info["team_size"]
```

Comment:
In case it changed in the database. What if I accidentally set team_size to the wrong number? The server will pull that data initially but not update it later? That makes no sense.